caledonia/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
```

Comment:
This is a basic question, but should we remove the `caledonia/` folder as it's no longer needed: project-root/
```
├── benches/
├── examples/ 
├── src/
```